pom.xml

```diff
@@ -271,16 +277,6 @@
 				<groupId>org.apache.kafka</groupId>
 				<artifactId>kafka_2.10</artifactId>
 				<version>0.8.2.2</version>
-				<exclusions>
```

Comment:
We should be able to keep those exclusions. In the end, we choose at packaging time which logger we inject (log4j-over-slf4j or log4j proper). Same for other exclusions of logging. I'm still hoping to get rid of those log4j loggers and of log4j one of theses days ;-)